tests/config.yml

```diff
@@ -9,6 +9,8 @@ monolog:
             level: info
 cache:
     provider: cache.service
+    ignore_namespaces:
+        - 'Symfony\\'
 
```

Comment:
Maybe we can also write some tests around this? With this change we would reach coverage, but the functionality around ignored namespaces might be broken...